src/Swift.Runtime/src/Library/ARC.cs

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Swift.Runtime;
+
+/// <summary>
+/// ARC is a class containing p/invokes for Swift reference counting and memory management.
+/// </summary>
+public static class ARC {
```

Comment:
If we go with the abbreviation, the capitalization should be `Arc` to conform with [.NET Design Guidelines](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers).